src/lib/Tests/Form/DataMapper/LanguageCreateMapperTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Tests\Form\DataMapper;
+
+use eZ\Publish\API\Repository\Values\Content\LanguageCreateStruct;
+use eZ\Publish\API\Repository\Values\Content\LocationCreateStruct;
+use EzSystems\EzPlatformAdminUi\Form\Data\Language\LanguageCreateData;
+use EzSystems\EzPlatformAdminUi\Form\Data\Language\LanguageDeleteData;
+use EzSystems\EzPlatformAdminUi\Form\DataMapper\LanguageCreateMapper;
+use PHPUnit\Framework\TestCase;
+use EzSystems\EzPlatformAdminUi\Exception\InvalidArgumentException;
+
+class LanguageCreateMapperTest extends TestCase
+{
+    /**
+     * @var LanguageCreateMapper
+     */
+    private $mapper;
+
+    public function setUp()
+    {
+        $this->mapper = new LanguageCreateMapper();
+    }
+
+    public function tearDown()
+    {
+        unset($this->mapper);
+    }
+
+    /**
+     * @dataProvider dataProvider
+     */
+    public function testMap(array $properties)
```

Comment:
To keep consistency I believe we should use docblocks for type hinting in tests as well.